sleap/nn/inference.py

```diff
@@ -2622,6 +2622,7 @@ def _object_builder():
                         # Set tracks for predicted instances in this frame.
                         predicted_instances = self.tracker.track(
                             untracked_instances=predicted_instances,
+                            img_hw=ex["image"].shape[-3:-1],
```

Comment:
From group review: Verify that channels are always last

Reply:
Verified!

In logs:
`INFO:sleap.nn.training:  Input shape: (192, 192, 1)`